### DISCUSSION TITLE
[FEATURE REQUEST] OPUS audio support

### DISCUSSION THREAD

**Discussion by knuxyl** [raysan5/raylib#2624] [Features]
OPUS is the latest open source audio codec that provides much better quality and lower bitrates than all competitors. It would be great to have this implemented as an alternative to OGG. This would help in distributing smaller builds due to smaller file sizes.

I looked through the OGG decoder and this is out of my capabilities atm.

**Comment by raysan5** [comment]
@knuxyl do you know any single-file header-only library that loads OPUS format? raylib only uses single-file header-only libraries for file-formats integration with the library.

**Comment by knuxyl** [comment]
There doesn't appear to be one yet. Maybe I'll look into once I am more proficient with C.
